### ISSUE TITLE
Tree Grid not available on extensions

### ISSUE THREAD

**Issue by pmorelli92** [wenzhixin/bootstrap-table#3606]
I was seeing the examples and the one using [TreeGrid ](http://issues.wenzhixin.net.cn/bootstrap-table/extensions/treegrid.html) is excellent. But there is a problem, the script:

`node-modules/bootstrap-table/src/extensions/treegrid/bootstrap-table-treegrid.js`

Is nowhere to be found **when installing with npm without indicating the source**
Am I the only one having this problem?

I can workaround the problem (for now) using:

`npm install git://github.com/wenzhixin/bootstrap-table.git#develop`

Regards,
Pablo

**Comment by jimgit** [comment]
One thing you could do is use the [link from the example ](http://issues.wenzhixin.net.cn/bootstrap-table/assets/bootstrap-table/src/extensions/treegrid/bootstrap-table-treegrid.js) http://issues.wenzhixin.net.cn/bootstrap-table/assets/bootstrap-table/src/extensions/treegrid/bootstrap-table-treegrid.jsbut probably better if you copy it to your server or cdn.

![image](https://user-images.githubusercontent.com/1921941/36311704-be324220-12f1-11e8-9731-9a0dcf95de61.png)

**Comment by pmorelli92** [comment]
@jimgit the problem is that for the projects we only reference third party scripts using npm to the point that if the npm package is wrong we cannot use it.

For now, as I described in the first post, I made a workaround getting the npm package directly from the git source instead from the published one. So now my package.json is:

`    "bootstrap": "^3.3.7",
    "bootstrap-notify": "^3.1.3",
    "bootstrap-table": "git://github.com/wenzhixin/bootstrap-table.git#develop",`

But this is not the ideal case.
